.github/workflows/notebooks-digest-updater-upstream.yaml

```diff
@@ -119,7 +119,7 @@ jobs:
         with:
           ref: ${{ env.DIGEST_UPDATER_BRANCH }}
 
-      - name: Fetch digest, and update the param.env file
+      - name: Fetch digest, and update the params.env file
```

Comment:
Since you're fixing this, would you mind to fix `param.env` also for the other section - line 73?